content/chapters/io/lab/support/async/python/client_bench.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+NUM_CLIENTS=8
+
+for i in $(seq 1 $NUM_CLIENTS); do
+    ./client.py localhost 2999 30 &
+done
```

Comment:
```suggestion
if [[ $# != 1 ]]; then
    echo "usage: $0 <port>"
    exit 1
fi

NUM_CLIENTS=8
PORT=$1

for i in $(seq 1 $NUM_CLIENTS); do
    ./client.py localhost "$1" 30 &
done
```
Take `PORT` as command line argument.

Reply:
Not really needed. Port 2990 is not that common. Maybe if it were 6969...